services/contract_create.proto

```diff
@@ -177,4 +178,14 @@ message ContractCreateTransactionBody {
      * cover auto-renewal fees.
      */
     AccountID auto_renew_account_id = 15;
+
+    /**
+     * ID of the new account to which this contract is staking. If not staked to an account, the value is null. The default value is null.
+     */
+    AccountID staked_account_id = 17;
+
+    /**
+     * If true, the contract declines receiving a staking reward. The default value is false.
+     */
+    google.protobuf.BoolValue decline_reward = 18;
```

Comment:
This should just be a primitive boolean on creates. Only updates need a wrapper. Same comment for other creates.

Reply:
Fixed. Thanks @steven-sheehy